Dockerfile.test-tools

```diff
@@ -0,0 +1,24 @@
+# based on build-tools but adds postgresql-client and postgresql packages
```

Comment:
I probably wouldn't limit us only by postgresql-client and postgresql only.
For example, we also could move docker to this image:
https://github.com/neondatabase/neon/blob/bd2ac822eb9a536af0f58164e5ba8aa606ddc692/Dockerfile.build-tools#L83-L88